google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonToProtoMessage.java

```diff
@@ -102,8 +99,10 @@ public class JsonToProtoMessage implements ToProtoConverter<Object> {
           .optionalEnd()
           .toFormatter()
           .withZone(ZoneOffset.UTC);
+  private final String ROOT_JSON_SCOPE = "root";
+  private final boolean DONT_ACCEPT_UNKNOWN_FIELDS = false;
```

Comment:
It is a bit strange to have this constant. I think it is easier to read just to keep the bollean.

Reply:
Are you sure? It explicitly names the intention. Otherwise you may mix boolean values:
`return convertToProtoMessage(protoSchema, null, json, "root", true, false);`
Can you tell what is `true` and `false` here? Your IDE will probably help you here, but what about GitHub review? Are you sure you won't put `topLevel` value in place of `ignoreUnknownArguments` parameter?

Reply:
Usually we will just annotate the parameters with the argument name. So it is just:
 protoSchema,
        tableSchema.getFieldsList(),
        json,
        "root",
        /*topLevel=*/ true,
        /*ignoreUnknownFields*/ false);

Reply:
Do you want me to inline only `DONT_ACCEPT_UNKNOWN_FIELDS` or `ROOT_JSON_SCOPE` too?
What IDE do you use? I thought all of them display the names of the parameters where constants are used. See the screenshot:
![Screenshot 2023-04-27 at 19 30 11](https://user-images.githubusercontent.com/8767916/234942414-e6a75f36-fd74-4857-a18c-4b322d6d6c25.png)

Reply:
DONT_ACCEPT_UNKNOWN_FIELDS and ROOT_JSON_SCOPE. Thanks.